en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -215,7 +215,7 @@ If we assign to an index beyond the current length, **the array will be extended
 
 #### Making a distinction between objects and arrays
 
-Both arrays and objects have the `'object'` type, therefore it is necessary to use the [`Array.isArray()`]( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) method to tell them apart.

```

Comment:
This one gave me a slight headache, as it isn't just a matter of the craft but also a bit of a philosophical quandary. Does a thing possess type as a property, or does type possess the thing? I've talked it out with Salva (these are things we often discuss, usually over beer) and while in the specific case of JavaScript, type is "a thing objects have", we agree it's both more philosophically intuitive, and representative of object-oriented languages in general, to have objects belong to types. So, approved.